### PR TITLE
[HE] difficulty

### PR DESCRIPTION
**Pull request by davidpodless**
Hi, long time no see....

he-39
he-40

### CODE REVIEW

**Review by qwrtln — CHANGES_REQUESTED** [review]
Please adjust the table cell heights. You can clearly see that the last one is too high, and the one before is too short.

**Comment by davidpodless** [comment]
> Please adjust the table cell heights. You can clearly see that the last one is too high, and the one before is too short.

How do I do it?

**Comment by davidpodless** [comment]
what do you think?

he-39
he-40

**Review by qwrtln — APPROVED** [review]

File: translations/difficulty.tex/he.po [hunk 1]
```diff
@@ -127,6 +142,40 @@ msgid ""
...
+"אחד במהלך מתקפה.}} \\\\ \\darkcell[0.8]{\\texthebrew{יורדת}} & "
+"\\lightcell[0.8]{\\texthebrew{כל המכרות והיישובים מספקים הכנסה כפולה.}} \\\\ "
+"\\darkcell[1.2]{\\texthebrew{יורדת}} & \\lightcell[1.2]{\\texthebrew{ניתן "
+"להחליף את המשאבים שלך בכל רגע, עמדות המשחק הופכות \\textbf{לניתנות לביקור} "
+"ומאפשרות לך לשלוף קלף 1 מחפיסת החפצים.}} \\\\ \\darkcell[0.8]"
+"{\\texthebrew{יורדת}} & \\lightcell[0.8]{\\texthebrew{המשך הקרב לא דורש עוד "
+"נקודות תנועה.}} \\\\ \\darkcell[0.8]{\\texthebrew{שונה}} & \\lightcell[0.8]"
+"{\\texthebrew{קובית ההתקפה לא משפיעה על הנזק  (אך עדיין בשימוש עם יכולות).}} "
+"\\\\ \\darkcell[0.8]{\\texthebrew{שונה}} & \\lightcell[0.8]"
+"{\\texthebrew{קלפי אסטרולוגיה נשלפים גם בתחילת סבב משאבים .}} \\\\ "
```
Comment (qwrtln) [inline translations/difficulty.tex/he.po:171]:
```suggestion
"{\\texthebrew{קלפי אסטרולוגיה נשלפים גם בתחילת סבב משאבים.}} \\\\ "
```

**Pull request merged**